types/express-http-proxy/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                  Gabriel Fournier <https://github.com/carboneater>
 //                  Niek van Bennekom <https://github.com/niekvb>
 //                  John L. Singleton <https://github.com/jsinglet>
+//                  Linday Wardell <https://github.com/lindsaykwardell>
```

Comment:
Typo in "Linday"?

Reply:
Whoops, thank you! Just pushed a correction.